packages/rollup-config-react-component/README.md

```diff
@@ -0,0 +1,40 @@
+# `@nl-design-system/rollup-config-react-component`
+
+A shareable [Rollup](https://rollupjs.org) configuration for React components.
+
+## Installation
+
+```shell
+$ npm install --save-dev @nl-design-system/rollup-config-react-component rollup
+# or
+$ yarn add --save-dev @nl-design-system/rollup-config-react-component rollup
+# or
+$ pnpm add --save-dev @nl-design-system/rollup-config-react-component rollup
+```
```

Comment:
Ik denk dat het handiger is om losse code blocks te doen met statements die je 1-op-1 kan kopieëren.